src/web/app/components/sessions-recycle-bin-table/sessions-recycle-bin-table.component.html

```diff
@@ -75,14 +75,16 @@ <h2 class="text-muted">
                 {{ recycleBinFeedbackSessionRowModel.feedbackSession.deletedAtTimestamp | recycleBinTableFormatDate:recycleBinFeedbackSessionRowModel.feedbackSession.timeZone }}
               </span>
             </td>
-            <td class="text-center actions-cell align-row">
-              <button id="btn-restore-{{ i }}" type="button" class="btn btn-light btn-sm" (click)="restoreSessionEvent.emit(recycleBinFeedbackSessionRowModel)">Restore</button>
-              <button id="btn-delete-{{ i }}" type="button" class="btn btn-light btn-sm text-danger align-row" (click)="permanentlyDeleteSessionEvent.emit(recycleBinFeedbackSessionRowModel); setRowClicked(i)">
-                <span>Delete Permanently</span>
-                <div class="padding-left-7px">
-                  <tm-ajax-loading [color]="'#000000'" *ngIf="isPermanentDeleteLoading && rowClicked === i"></tm-ajax-loading>
-                </div>
-              </button>
+            <td class="text-center actions-cell">
+              <div class="d-flex">
+                <button id="btn-restore-{{ i }}" type="button" class="btn btn-light btn-sm" (click)="restoreSessionEvent.emit(recycleBinFeedbackSessionRowModel)">Restore</button>
+                <button id="btn-delete-{{ i }}" type="button" class="btn btn-light btn-sm text-danger align-row" (click)="permanentlyDeleteSessionEvent.emit(recycleBinFeedbackSessionRowModel); setRowClicked(i)">
```

Comment:
I think we can remove the `align-row` class here, since it adds a redundant padding inside the button and causes the text to not be centered.

<img width="719" alt="Screen Shot 2023-05-02 at 8 47 30 AM" src="https://user-images.githubusercontent.com/25262042/235556272-b5a486d7-9490-4664-87e8-252589999630.png">

Reply:
By doing so, we can also delete the `align-row` CSS class in the SCSS file

Reply:
Thank you for letting me know. I have made the changes according to your review. The updated UI does not have the margin now: 
![截屏2023-05-01 下午9 19 31](https://user-images.githubusercontent.com/41343828/235559501-33c976f8-8d4a-4169-a0ab-47da188bf9e6.png)